WordPress/gutenberg
# [RNMobile] - Replace Slider with Stepper for columns settings on Gallery block

`Gutenberg Mobile` PR -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/1834

## Description
This PR replaces the `RangeControl` that handles the number of columns for the `Gallery` block with the [Stepper component](https://github.com/WordPress/gutenberg/pull/18864) **only** for mobile.

It also changes the prop names of the `Stepper` component to match the `RangeControl` for simpler use. Since this component is not being used until now it was the best time to do it.

| iOS | Android |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/73362691-93828e80-42a7-11ea-8521-0e2817dfd2aa.gif" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/73362696-95e4e880-42a7-11ea-83bb-ac8d61c7d5a8.gif" width=250 />  |

## How has this been tested?
- Open the app with metro running
- Create a post
- Add a `Gallery` block
- Select some images
- Open the block settings
- **Expect**: See the new Stepper component under the Columns setting
- Change the value and check it changes correctly

## Screenshots 
| iOS | Android |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/73362423-19520a00-42a7-11ea-8e3f-e6d83854e082.png" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/73362430-1e16be00-42a7-11ea-9321-0981bffe0542.png" width=250 />  |

## Types of changes
- Replace the mobile component for another one.
- Update the `Stepper` component props name to match the `RangeControl`

## Checklist:
